src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js

```diff
@@ -109,7 +109,7 @@ const VersionsToolbar = ({
                 }
                 secondary={
                   <div style={{ color: 'black' }}>
-                    {option.version.effectiveLegalEnd == null
+                    {option.version.iscurrentlegalversion === true
```

Comment:
@micheal-w-wells Would it be possible to make this field camelCase like the others? It's probably a backend change, but I'm not sure how difficult it would be.

Reply:
I'll make a point of doing so in next change to the migration, it is just a table alias so not an issue to change